linux/guest_customization/check_gosc_support_status.yml

```diff
@@ -126,7 +131,6 @@
       when:
         - not enable_cloudinit_gosc | bool
         - guest_os_ansible_distribution not in gos_not_support_gosc
```

Comment:
Or use gosc_is_supported directly.

Reply:
We have a rule which is that is if the var's value is changed in block, do not use it in when conditions of the outer block.